taxonomies/ingredients.txt

```diff
@@ -23798,6 +23798,9 @@ de:natürliches Dill-Aroma, natürliches Dillaroma
 fr:arôme naturel d'aneth
 # ingredient/fr:arome-naturel-d'aneth has 34 products in 1 language @2020-05-21
 
+<en:flavouring
+es:sabor idéntico al natural pollo
```

Comment:
According to the legal definition, this is an artificial chicken flavouring that has the same molecules as the ones that can be found in chicken: http://www.sice.oas.org/trade/mrcsrs/resolutions/an4693.asp

So we can put it under <en:chicken flavouring

The English term is "Nature-identical flavour".

So we can create an entry "Nature-identical chicken flavour"

I think it's best to try to always add an English name for all entries we add.